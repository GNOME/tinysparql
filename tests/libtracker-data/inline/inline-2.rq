SELECT ?a ?b {
  VALUES (?a ?b) {
    ( 1 'a' )
    ( 'b' 2 )
    ( 3 'c' )
    ( UNDEF 4.2 )
  }
}
