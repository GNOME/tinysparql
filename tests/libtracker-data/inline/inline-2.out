"1"	"a"
"b"	"2"
"3"	"c"
	"4.2"
